apps/backend/src/build-notification/comment.ts

```diff
@@ -3,42 +3,69 @@ import { invariant } from "@argos/util/invariant";
 import { getBuildLabel } from "@/build/label.js";
 import { getStatsMessage } from "@/build/stats.js";
 import { getCommentHeader } from "@/database/index.js";
-import { Build } from "@/database/models/index.js";
+import { Build, Project } from "@/database/models/index.js";
 
 const dateFormatter = Intl.DateTimeFormat("en-US", {
   dateStyle: "medium",
   timeStyle: "short",
   timeZone: "UTC",
 });
-const formatDate = (date: string): string => {
+
+function formatDate(date: string): string {
   const d = new Date(date);
   return dateFormatter.format(d);
-};
+}
+
+/**
+ * Get the name of the build.
+ * If the build is associated with a project, the project name is prepended.
+ */
+function getBuildName(input: { build: Build; project: Project | null }) {
+  if (input.project) {
+    return `${input.project.name}/${input.build.name}`;
+  }
+
+  return input.build.name;
+}
 
 export const getCommentBody = async (props: {
   commit: string;
 }): Promise<string> => {
   const builds = await Build.query()
-    .distinctOn("builds.name")
+    .distinctOn("builds.name", "builds.projectId")
     .joinRelated("compareScreenshotBucket")
+    .withGraphFetched("project")
     .where("compareScreenshotBucket.commit", props.commit)
     .orderBy([
-      { column: "name", order: "desc" },
+      { column: "builds.name", order: "desc" },
+      { column: "builds.projectId", order: "desc" },
       { column: "builds.number", order: "desc" },
     ]);
 
```

Comment:
To remove

Reply:
Why, no if it's in distinctOn it's required.

Reply:
If you talk about console log it's done.